genologics/lims.py

```diff
@@ -18,7 +18,7 @@
 # python 2.7, 3+ compatibility
 from sys import version_info
 
-if version_info.major == 2:
```

Comment:
I kinda prefer `major` over `[0]` as it's clearer. Is there a specific reason to use [0] ?